src/app/vendors/SingleVendor.tsx

```diff
@@ -165,6 +165,7 @@ function SingleVendor({
 
   let displayName = d2Vendor.def.displayProperties.name;
   let displayDesc = d2Vendor.def.displayProperties.description;
+  let artifactCheck = 'notArtifact';
```

Comment:
this could just as easily be
`let artifactCheck:string|undefined;`
there's no use for the notartifact string

(or maybe
`let artifactCheck:string;`
or even
`let artifactCheck;`
depending on how much ts is willing to situationally infer)

Reply:
yeah I had an empty string there but I filled it in cause i use to as an ID later so this prevents an empty id attribute later. I don't mind going back to string empty though.

Reply:
not an empty sting, `undefined`